src/Hotel.js

```diff
@@ -1,12 +1,46 @@
+import moment from 'moment';
+
 class Hotel {
-  constructor(users, rooms, bookings) {
-    this.users = this.filterInvalidData(users);
-    this.rooms = this.filterInvalidData(rooms);
-    this.bookings = this.filterInvalidData(bookings);
+  constructor(rooms, bookings, date) {
```

Comment:
I'm instantiating this with today's date instead of the most recent one in the data - wondering how you were handling this.

Reply:
I'm handling it the same - although I do think you'll want to access the `date` value again later on. Might want to store it as a property.

Reply:
Same idea with your comment down below about `currentUser` - I'll definitely make a point to get this info to persist